Automattic/wp-calypso
# DevDocs: Fix missing display names of some components.

Resolves #19343 

## How to test
1. Run Calypso with `wpcalypso` env:
```
$> CALSYPSO_ENV="wpcalypso" npm start
```
2. Check out `/devdocs/design` to see if `Checklist`, `Notices`, `PaymentLogo` and `Rating` properly display its name rather than `t`.
